lambdas/process-async-cri-credential/src/main/java/uk/gov/di/ipv/core/processasynccricredential/ProcessAsyncCriCredentialHandler.java

```diff
@@ -167,44 +168,49 @@ private void processErrorAsyncCriResponse(ErrorAsyncCriResponse errorAsyncCriRes
                         .with(
                                 LOG_ERROR_DESCRIPTION.getFieldName(),
                                 errorAsyncCriResponse.getErrorDescription())
-                        .with(LOG_ERROR_CODE.getFieldName(), errorAsyncCriResponse.getError())
-                        .with(
-                                LOG_CRI_ISSUER.getFieldName(),
-                                errorAsyncCriResponse.getCredentialIssuer()));
+                        .with(LOG_ERROR_CODE.getFieldName(), errorAsyncCriResponse.getError()));
 
-        sendIpvVcErrorAuditEvent(errorAsyncCriResponse);
+        if (criResponseItem.isPresent()
```

Comment:
This will only be called when there is a cri response, as we need it to check for F2F cri Id.